l10n_vn_account_invoice_number/models/account_invoice.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Trobz (http://trobz.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+
+
+class AccountInvoice(models.Model):
+    _inherit = "account.invoice"
+
+    tax_invoice_number = fields.Integer()
+    tax_invoice_form = fields.Char()
+    tax_invoice_serie = fields.Char()
+    is_owner_in_vn = fields.Boolean(string='Is Owner in Vietnam',
+                                    store=False,
+                                    compute='_compute_is_owner_in_vn')
+
+    @api.onchange('partner_id', 'company_id')
+    def _onchange_partner_id(self):
+        self._compute_is_owner_in_vn()
+        return super(AccountInvoice, self)._onchange_partner_id()
+
+    @api.multi
+    def _compute_is_owner_in_vn(self):
```

Comment:
I think it could be nicer if renaming to `_compute_is_issuer_in_vn` and same for variables and field name